src/Chapter2.hs

```diff
@@ -501,8 +516,9 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
-
+isThird42 :: [Int] -> Bool
+isThird42 (_:_:a:_) = (a == 42)
```

Comment:
You don't need to compare `z` with 42 separately, you can pattern match on `42` directly in the pattern.

```suggestion
isThird42 (_:_:42:_) = True
```